.github/styles/Vocab/Docs/accept.txt

```diff
@@ -226,6 +226,9 @@ Snapshotter
 socat
 softmax
 Splunk
+(Statefulset|statefulset)
+statefulset(|s)
+Statefulset(|s)
```

Comment:
This probably does the same thing, but it's just a nit - nice cleanup @gvasquezvargas 

```suggestion
statefulset(|s)
Statefulset(|s)
```